htmlgen/htmlgen.py

```diff
@@ -999,8 +1003,7 @@ def main():
     parser.add_argument('--verbose', action='store_true')
     parser.add_argument(
         '--output',
-        default=os.path.join(
-            os.path.curdir, 'html', get_setting('XRAY_DATABASE')),
+        default=os.path.join(os.path.curdir, 'html'),
```

Comment:
It would be better if this defaulted to `None` and then when it was `None` you set it to `os.path.curdir, 'html', get_setting('XRAY_DATABASE')`